geode-web/src/test/java/org/apache/geode/management/internal/web/controllers/support/LoginHandlerInterceptorJUnitTest.java

```diff
@@ -89,52 +79,66 @@ public T nextElement() {
+    when(mockHttpRequest.getHeader(PASSWORD)).thenReturn("password");
+    when(mockHttpRequest.getParameter(ENVIRONMENT_VARIABLE_REQUEST_PARAMETER_PREFIX + "variable"))
+        .thenReturn("two");
 
     LoginHandlerInterceptor handlerInterceptor = new LoginHandlerInterceptor(securityService);
+    Map<String, String> environmentBeforePreHandle = LoginHandlerInterceptor.getEnvironment();
+    assertThat(environmentBeforePreHandle).isNotNull();
+    assertThat(environmentBeforePreHandle.isEmpty()).isTrue();
+
+    assertThat(handlerInterceptor.preHandle(mockHttpRequest, null, null)).isTrue();
+    Map<String, String> environmentAfterPreHandle = LoginHandlerInterceptor.getEnvironment();
+    assertThat(environmentAfterPreHandle).isNotNull();
+    assertThat(environmentBeforePreHandle).isNotSameAs(environmentAfterPreHandle);
+    assertThat(1).isEqualTo(environmentAfterPreHandle.size());
+    assertThat(environmentAfterPreHandle.containsKey("variable")).isTrue();
+    assertThat("two").isEqualTo(environmentAfterPreHandle.get("variable"));
+    Properties expectedLoginProperties = new Properties();
+    expectedLoginProperties.put(USER_NAME, "admin");
+    expectedLoginProperties.put(PASSWORD, "password");
+    verify(securityService, times(1)).login(expectedLoginProperties);
+  }
 
-    Map<String, String> envBefore = LoginHandlerInterceptor.getEnvironment();
-
-    assertNotNull(envBefore);
-    assertTrue(envBefore.isEmpty());
-    assertTrue(handlerInterceptor.preHandle(mockHttpRequest, null, null));
-
-    Map<String, String> envSet = LoginHandlerInterceptor.getEnvironment();
+  @Test
+  public void afterCompletionShouldCleanTheEnvironment() throws Exception {
+    final Map<String, String> requestParameters = new HashMap<>(2);
+    requestParameters.put(ENVIRONMENT_VARIABLE_REQUEST_PARAMETER_PREFIX + "variable", "two");
+    final HttpServletRequest mockHttpRequest = mock(HttpServletRequest.class, name.getMethodName());
+    when(mockHttpRequest.getParameterNames())
+        .thenReturn(enumeration(requestParameters.keySet().iterator()));
+    when(mockHttpRequest.getHeader(USER_NAME)).thenReturn("admin");
+    when(mockHttpRequest.getHeader(PASSWORD)).thenReturn("password");
+    when(mockHttpRequest.getParameter(ENVIRONMENT_VARIABLE_REQUEST_PARAMETER_PREFIX + "variable"))
+        .thenReturn("two");
 
-    assertNotNull(envSet);
-    assertNotSame(envBefore, envSet);
-    assertEquals(1, envSet.size());
-    assertTrue(envSet.containsKey("variable"));
-    assertEquals("two", envSet.get("variable"));
+    LoginHandlerInterceptor handlerInterceptor = new LoginHandlerInterceptor(securityService);
+    assertThat(handlerInterceptor.preHandle(mockHttpRequest, null, null)).isTrue();
+    Map<String, String> environmentAfterPreHandle = LoginHandlerInterceptor.getEnvironment();
+    assertThat(environmentAfterPreHandle).isNotNull();
+    assertThat(1).isEqualTo(environmentAfterPreHandle.size());
```

Comment:
Looking at the rest of this test class, all of your `assertThat(<actual>).is*(expected)` have the `actual` and `expected` values swapped. Please review the rest of the tests were refactored to use assertj for similar swapping of values.